grafana-plugin/src/containers/RotationForm/parts/DateTimePicker.tsx

```diff
@@ -73,7 +88,7 @@ export const DateTimePicker = observer(
             <DatePickerWithInput
               open
               disabled={disabled}
-              value={getDateForDatePicker(valueInSelectedTimezone)}
```

Comment:
I guess getDateForDatePicker is no longer used? Can we remove it in this case?

Reply:
yes, I have it removed in a stash. It will be removed.